core/drivers/stm32_tamp.c

```diff
@@ -133,31 +161,184 @@ struct stm32_tamp_compat {
 };
 
 /* Expects at most a single instance */
-static struct stm32_tamp_instance *stm32_tamp_device;
+static struct stm32_tamp_instance *stm32_tamp_dev;
+
+static void apply_rif_config(void)
+{
+	vaddr_t base = io_pa_or_va(&stm32_tamp_dev->pdata.base, 1);
+	uint32_t seccfgr = 0;
+	uint32_t privcfgr = 0;
+	uint32_t access_mask_sec_reg = 0;
+	uint32_t access_mask_priv_reg = 0;
+	unsigned int i = 0;
```

Comment:
what about `struct rif_conf_data *rif_conf = stm32_tamp_dev->pdata.conf_data;`. It would prevent some line escapes in this function.